idseq_dag/steps/generate_phylo_tree.py

```diff
@@ -298,7 +298,7 @@ def get_accession_sequences(self, dest_dir, taxid, n=10):
                 local_basename = s3_file.replace("/", "-").replace(":", "-")
                 local_file = s3.fetch_from_s3(
                     s3_file,
-                    os.path.join(self.ref_dir_local, local_basename))
+                    os.path.join(self.output_dir_local, local_basename))
```

Comment:
This is a bug fix, we don't want these downloads persisted across jobs in the references directory.